eth/downloader/downloader.go

```diff
@@ -114,21 +114,11 @@ type Downloader struct {
 	syncStatsState       stateSyncStats
 	syncStatsLock        sync.RWMutex // Lock protecting the sync stats fields
 
+	lightchain LightChain
+	chain      BlockChain
```

Comment:
Lets call this either `blockchain` or `fullchain`. The plain `chain` is a bit misleading with the presence of `lightchain` imho,